clang/lib/AST/StmtPrinter.cpp

```diff
@@ -2566,6 +2566,11 @@ void StmtPrinter::VisitCXXUnresolvedConstructExpr(
     OS << ')';
 }
 
+void StmtPrinter::VisitCXXReflectExpr(CXXReflectExpr *S) {
+  // TODO: Make this better.
+  OS << "^(...)";
```

Comment:
Lol now might be the time to make it better 😂 

If we want to punt on printing for now, we should at least use `^^` instead of `^`.